test/runtime/samples/event-handler-dynamic/_config.js

```diff
@@ -14,8 +14,14 @@ export default {
 		);
 
 		const event = new window.MouseEvent('click');
+		let err = "";
+		window.addEventListener('error', (e) => {
+			e.preventDefault();
+			err = e.message;
```

Comment:
```suggestion
			err += e.message;
```

should concat all the error messages

Reply:
Thanks for the review, I'll push up changes after the holiday or feel free to push a commit sooner if needed.